test/inductor/test_torchinductor.py

```diff
@@ -3043,8 +3084,8 @@ def forward(arg6, arg7, arg16):
             # Mismatched elements: 127 / 746496 (0.0%)
             # Greatest absolute difference: 0.0009765625 at index (1, 62, 7, 16) (up to 1e-05 allowed)
             # Greatest relative difference: 0.05187467899332306 at index (14, 18, 11, 0) (up to 0.001 allowed)
-            atol=1e-3,
-            rtol=0.001,
+            atol=3e-3,
+            rtol=2,
```

Comment:
rtol=2 seems pretty bad... what is going on here?

Reply:
Sometimes channels last kernel can produce very different numerics especially with random input data. I'll see if I can repro this with eager channels last permutation.

Reply:
You get similarly high different rtol just by comparing eager channels last and eager contiguous, and with just the convolution:

> Mismatched elements: 3084650 / 3097600 (99.6%)
Greatest absolute difference: 0.02942657470703125 at index (7, 19, 34, 22) (up to 1e-05 allowed)
Greatest relative difference: 361.1468811035156 at index (3, 20, 48, 11) (up to 1.3e-06 allowed)